distribution/src/main/assembly/assembly.xml

```diff
@@ -28,7 +28,7 @@
         </fileSet>
         <fileSet>
             <directory>../resources/conf</directory>
-            <outputDirectory>WSO2-Microgateway-${project.version}/resources/pilot/conf</outputDirectory>
+            <outputDirectory>WSO2-Microgateway-${project.version}/resources/conf</outputDirectory>
```

Comment:
Lets use the existing distribution name we used for microgateway runtime

Reply:
Fixed with 2921e00